docs/templates/index.md

```diff
@@ -0,0 +1,199 @@
+
+### GET /v1/tsg/templates
+
+To list all of the templates associated with a specific Triton account, send a `GET` request to `/v1/tsg/templates` with
+the request headers detailed below.
+
+#### Example Request
+
+```
+curl -X GET "https://tsg.us-sw-1.svc.joyent.zone/v1/tsg/templates"
+```
+
+
+#### Request Headers 
+
+```
+Authorization: Signature keyId="/test-user-name/keys/aa:bb:cc:dd:9c:54:e9:78:3f:80:0d:ba:6b:c6:ff:44",algorithm="rsa-sha1",headers="date",signature="..."
+Date: Fri, 06 Apr 2018 18:33:38 UTC
+```
+
+#### Sample Response
+
+```
+[
+    {
+        "id": "437c560d-b1a9-4dae-b3b3-6dbabb7d23a7",
+        "template_name": "test-template-6",
+        "account_id": "6f873d02-172c-418f-8416-4da2b50d5c53",
+        "package": "test-package",
+        "image_id": "49b22aec-0c8a-11e6-8807-a3eb4db576ba",
+        "firewall_enabled": false,
+        "networks": [
+            "f7ed95d3-faaf-43ef-9346-15644403b963"
+        ],
+        "metadata": {
+    	    "root_pw": "s8v9kuht5e"
+    	}, 
+        "tags": {
+            "role": "web",
+            "owner": "api-team"
+        }
+    }
+}
+```
+
+### GET /v1/tsg/templates/{identifier}
+
+To show information about a specific template, send a `GET` request to `/v1/tsg/templates/{identifier}` 
+using the request headers as detailed below.
+ 
+
+#### Example Request
+
+```
+curl -X GET "https://tsg.us-sw-1.svc.joyent.zone/v1/tsg/templates/319209784155176962"
+```
+
+#### Request Headers 
+
+```
+Authorization: Signature keyId="/test-user-name/keys/aa:bb:cc:dd:9c:54:e9:78:3f:80:0d:ba:6b:c6:ff:44",algorithm="rsa-sha1",headers="date",signature="..."
+Date: Fri, 06 Apr 2018 18:33:38 UTC
+```
+
+#### Sample Response
+
+```
+{
+    "id": "437c560d-b1a9-4dae-b3b3-6dbabb7d23a7",
+    "template_name": "test-template-6",
+    "account_id": "6f873d02-172c-418f-8416-4da2b50d5c53",
+    "package": "test-package",
+    "image_id": "49b22aec-0c8a-11e6-8807-a3eb4db576ba",
+    "firewall_enabled": false,
+    "networks": [
+        "f7ed95d3-faaf-43ef-9346-15644403b963"
+    ],
+    "metadata": {
+	    "root_pw": "s8v9kuht5e"
+	}, 
+    "tags": {
+        "role": "web",
+        "owner": "api-team"
+    }
+}
+```
+
+### POST /v1/tsg/templates
+
+To create a new template, send a `POST` request to `/v1/tsg/templates`. The
+request needs to include the headers as identified below. A successful creation will return 
+a `201 created` HTTP Response Code. The attributes required to successfully create a template are as follows:
+
+
+ | Name             | Type              | Required  |
+ |:----------:      |:-------------:    |:------:   |
+ | Template Name    | string            | true      |
+ | Package          | string            | true      |
+ | ImageID          | string            | true      |
+ | FirewallEnabled  | bool              |           |
+ | Networks         | []string          |           |
```

Comment:
Pardon my unfamiliarity with this, but can you explain "[]string" as a type? I'm familiar with strings, bools, arrays, etc and if I had to guess based on the body of the request it's because the string is contained in []. Is that commonly how the type would be declared?

Reply:
I have corrected these accordingly, and also made the field names into what we need them to be in the actual JSON payload/request.

Reply:
We just corrected the same thing here :)

Reply:
Sorry about that :(